doc/source/_static/css/pandas.css

```diff
@@ -36,17 +36,6 @@ table {
   padding: 2.5rem 0rem 0.5rem 0rem;
 }
 
-.intro-card .card-footer {
```

Comment:
I don't think we should modify the css for this. Am I missing something? I think you probably want to revert this deletion.